src/errors/scErrors.sol

```diff
@@ -21,3 +21,4 @@ error FlashLoanAmountZero();
 error ProtocolNotSupported(uint256 adapterId);
 error ProtocolInUse(uint256 adapterId);
 error FloatBalanceTooLow(uint256 actual, uint256 required);
+error TokenNotAllowed(address token);
```

Comment:
nitpicking but would be more informative to name it `TokenOutNotAllowed`

Reply:
Agree. [Changed](https://github.com/lindy-labs/sandclock-contracts/pull/120/commits/c62ab80b2a07e7215129dd106bafac6d891c31e9).